app/views/docs/offline.phtml

```diff
@@ -0,0 +1,20 @@
+<p>
+    The Appwrite SDKs' Offline Support allows your client apps to continue functioning without an internet connection. Your users' mobile devices frequently encounter situations where internet access becomes unstable or temporarily unavailable, like when going through tunnels, getting on airplanes and subways, or going in and out of Wi-Fi range. Offline Support helps your app gracefully handle these real-life situations and keeps your users engaged.
+</p>
+
+<h2>When Should I Enable Offline Support?</h2>
+<p>
+    With Offline Support enabled, your app can continue sending requests to Appwrite APIs while offline through a cache. While offline, read requests will be fetched from the local cache data, and write requests will be queued and executed once the device is online again.
```

Comment:
> read requests will be fetched from the local cache data

This sounds a little odd to me because the response is fetched from the local cached data rather than the request.

Also, maybe it should be:

```suggestion
    With Offline Support enabled, your app can continue sending requests to Appwrite APIs while offline through a cache. While offline, read requests will be fetched from the local cached data, and write requests will be queued and executed once the device is online again.
```